packages/react-microlink/src/components/Card/CardMedia/Video/controls.js

```diff
@@ -36,19 +41,17 @@ export const PlayButton = styled.div`
 export const ProgressBar = styled.div.attrs({
   style: ({playing, progress}) => ({
     width: `${progress}%` || 0,
-    opacity: playing ? 0.7 : 0
+    opacity: playing ? 0.8 : 0
   })
 })`
   position: absolute;
   left: 0;
   bottom: 0;
   background: #e1e8ed;
-  height: 2px;
-  border-top-right-radius: 2px;
-  transition: opacity 0.3s ease;
+  height: ${({cardSize}) => getSize(PROGRESS_BAR_HEIGHT, cardSize)}px;
+  transition: opacity 0.15s ease-in-out;
 
-  .microlink_card:not(:hover) & {
+  div.microlink_card:not(:hover) & {
```

Comment:
We can't add `div`, cause the card could be rendered as _any_ html element @Kikobeats